src/utils.js

```diff
@@ -1,8 +1,15 @@
 import b4a from 'b4a'
+import sodium from 'sodium-universal'
 import { keyToPublicId } from '@mapeo/crypto'
 import { createHash } from 'node:crypto'
 import stableStringify from 'json-stable-stringify'
 
+const PROJECT_INVITE_ID_SALT = Buffer.from(
```

Comment:
I have previously used human-readable words (as a buffer) for this. This is based on what I have seen Hypercore do. I _think_ it's just as secure, and perhaps easier to read / check it hasn't changed.

Reply:
Done in bc5714357f9d6e9259ae4d80e8045665ccbc2779.